### ISSUE TITLE
Display version number 

### ISSUE THREAD

**Issue by mzuther** [premake/premake-core#1095]
Hi!

Is there a way to force premake5 display its version number on the command line?  If not, I think it would be a useful feature ("Hmm, do I have the latest premake build?").  I can think of two ways to display it: either if premake is called without any files or options, or by calling "premake5 --version", whatever you like best.

Thanks,

Martin

**Comment by starkos** [comment]
`premake5 --version`

You’re welcome :)

**Comment by mzuther** [comment]
Hi starkos,

thanks for your help!  I was wondering why I could have been so stupid to not have tried the obvious.  It turns out that I actually did.  However, as I happened at the time to be in a directory with a faulty `premake5.lua`, I got an error when I tried `premake5 --help` and `premake5 --version`.  As this is unexpected behaviour, I consider it a bug in premake 5.0.0-alpha12.

To reproduce, create a minimal `premake5.lua` file with the following contents:

    if _ACTION == "gmake" then
        print ("gmake")
    end
    
    workspace "kmeter"
        location (os.target() .. "/" .. _ACTION .. "/")

If you are in the directory where this file resides, `premake5 gmake --help` works as expected, while `premake5 --help` only prints an ugly error ("attempt to concatenate a nil value"). Same for `premake5 --version`.
